notes/v1.3.0-RC1.md

```diff
@@ -1,3 +1,384 @@
+you pointers to help implement it.
+
+## Fix use of test arguments in `bloop test` :chart_with_upwards_trend:
+
+Running `bloop test` in your project or passing specific test arguments to
+Bloop's test runner could [produce issues such as this one](https://github.com/scalacenter/bloop/issues/906):
+
+```
+[E] Unexpected error during remote test execution: 'Socket closed'.
+[E] Internal error when running tests: sbt.ForkMain$Run$RunAborted: java.net.SocketException: Broken pipe (Write failed)
+```
+
+[This pull request](https://github.com/scalacenter/bloop/pull/843) makes sure
+that we handling missing test frameworks better and that the above issues
+processing test arguments are gone.
+
+## Improvements in build tools support (`bloopInstall`) :hammer_and_wrench:
+
+### Improvements in `sbt`
+
+#### Automatic support for integration tests in sbt builds
+
+`sbt-bloop` v1.3.0 will [automatically export integration test
+configurations](https://github.com/scalacenter/bloop/pull/873) to independent
+build targets. If you have modified your sbt build to support integration tests
+in your build, you can safely remove that logic.
+
+#### Support sbt source dependencies by default
+
+Contributed by [olafurpg]. Support for sbt project references was before
+hidden behind an specific sbt boolean setting. [Now it's enabled by
+default](https://github.com/scalacenter/bloop/pull/816).
+
+#### Other
+
+1. [Avoid duplication of projects in `dependencies` field](https://github.com/scalacenter/bloop/pull/883) by [tkroman]
+1. [Fix small bug when exporting scalac and javac options in sbt](https://github.com/scalacenter/bloop/pull/876) by [jvican]
+
+### Improvements in `Gradle`
+
+1. [Include Gradle's transitive dependencies in classpath](https://github.com/scalacenter/bloop/pull/818) by [Arthurm1]
+1. [Ignore annotation processor in Gradle](https://github.com/scalacenter/bloop/pull/878) by [tgodzik]
+1. [Don't add JAR dependencies of Gradle projects in the classpath](https://github.com/scalacenter/bloop/pull/875) by [tgodzik]
+1. [Export source dependencies from Gradle](https://github.com/scalacenter/bloop/pull/849) by [tgodzik]
+
+Thanks to these improvements, Metals will be able to support "Import Build"
+in Gradle projects in its next release.
+
+### Improvements in `Mill`
+
+1. [Point Mill docs to merged bloop integration upstream](https://github.com/scalacenter/bloop/pull/872) by [Baccata]
+1. [Fix broken link in Mill documentation](https://github.com/scalacenter/bloop/pull/828) by [perryao]
+1. [Update Mill version](https://github.com/scalacenter/bloop/pull/847) by [robby-phd]
+
+Thanks to these improvements, Metals will be able to support "Import Build"
+in mill builds in its next release.
+
+### Improvements in `Maven`
+
+1. [Avoid NPE when collecting resources in Maven's Bloop plugin](https://github.com/scalacenter/bloop/pull/839) by [artyomboyko]
+1. [Add source dependencies to Maven plugin](https://github.com/scalacenter/bloop/pull/888)
```

Comment:
Missing `by [tgodzik]` - that's not me being unhumble, but my OCD kicking in haha